css/repeating-radial-gradient/demos/basic/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Базовый пример — repeating-radial-gradient() — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+    .div {
```

Comment:
```suggestion
    div {
```

Лишняя точка. Селектор по тегу, а не по классу =)